### PR TITLE
blog: 1.0 release copy edits

### PR DESCRIPTION
**Pull request by jorgeorpinel**

### CODE REVIEW

File: content/blog/2020-06-22-dvc-1-0-release.md [hunk 1]
```diff
@@ -62,21 +62,21 @@ learned in 3 years of this journey and how these are reflected in the new DVC.
...
 pipelines with data processing steps. People need to change the commands of the
-pipeline often and it was not easy to do with the old DVC files.
+pipeline often and it was not easy to do this with the old DVC-files.
 
-In DVC 1.0, the DVC file format was changed in three big ways. First, instead of
-multiple DVC stage files (`*.dvc`), each project has a single DVC file
-`dvc.yaml`. By default, all stages go in this single `.yaml` file.
+In DVC 1.0, the DVC metafile format was changed in three big ways. First,
+instead of multiple DVC stage files (`*.dvc`), each project has a single
+`dvc.yaml` file. By default, all stages go in this single YAML file.
```
Comment (jorgeorpinel) [inline content/blog/2020-06-22-dvc-1-0-release.md:69]: > each project has a single `dvc.yaml` file

Actually you can have a dvc.yaml file in each subdirectory if you wanted. At least right now it's possible. Should that be mentioned @dmpetrov ?

**Comment by shcheklein** [comment]
@jorgeorpinel it's bit late to edit this post - let's not do this ... may be except typos ... but I would probably keep even them.

File: content/blog/2020-06-22-dvc-1-0-release.md [hunk 2]
```diff
@@ -105,47 +105,47 @@ stages:
...
-the result of the command from the cache without rerunning it. It is a useful
-feature for a hyperparameter optimization stage - when users return to the
+This approach gives DVC a "long memory" of DVC stages runs. If a user tries to
+run a stage that was previously run (whether committed to Git or not), then DVC
+can return the result from the run-cache without rerunning it. It is a useful
+feature for a hyperparameter optimization stage — when users return to the
 previous sets of the parameters and don't want to wait for ML retraining.
 
 Another benefit of the run-cache is related to CI/CD systems for ML, which is a
 holy grail of MLOps. The long memory means users don't have to make auto-commits
```
Comment (jorgeorpinel) [inline content/blog/2020-06-22-dvc-1-0-release.md:146]: Not sure what we mean by "auto-commits".

**Comment by jorgeorpinel** [comment]
OK @shcheklein. Yes, most of my changes here are for typos and small grammar issues.

**Comment by shcheklein** [comment]
is it still draft?

**Comment by jorgeorpinel** [comment]
It was draft because I only reviewed around half of it so far until you said this is unnecessary. Dmitry also said it's probably not needed anymore. Should we merge anyway? Up to you! Thanks

**Pull request merged**